AdvancedDLSupport/PathResolvers/WindowsPathResolver.cs

```diff
@@ -40,15 +40,15 @@ public ResolvePathResult Resolve(string library)
                 return ResolvePathResult.FromSuccess(libraryLocation);
             }
 
-            var windowsDir = Environment.GetFolderPath(Environment.SpecialFolder.Windows);
-
-            var sys32Dir = Path.Combine(windowsDir, "System32");
-            libraryLocation = Path.GetFullPath(Path.Combine(sys32Dir, library));
+            var sysDir = Environment.SystemDirectory;
+            libraryLocation = Path.GetFullPath(Path.Combine(sysDir, library));
             if (File.Exists(libraryLocation))
             {
                 return ResolvePathResult.FromSuccess(libraryLocation);
             }
 
+            var windowsDir = Environment.GetFolderPath(Environment.SpecialFolder.Windows);
+
```

Comment:
Spurious newline. Remove.

Reply:
I just followed the same style than before my changes, where there was a additional newline after windowsDir, but ok.